errors/arsenalErrors.json

```diff
@@ -711,5 +711,10 @@
   "NotEnoughMapsInConfig:": {
     "description": "NotEnoughMapsInConfig",
     "code": 400
+  },
+  "_comment": "----------------------- cdmiclient -----------------------",
+  "ReadOnly": {
+    "description": "read only back-end",
```

Comment:
"trying to write to ready only back-end" might be more descriptive?

Reply:
you re right, done.